.github/workflows/check-static.yml

```diff
@@ -0,0 +1,49 @@
+# Copyright (C) 2020  Matthew Glazar
+# See end of file for extended copyright information.
+
+name: static analysis
+on:
+  push:
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  analysis:
+    name: clang-tidy
+    runs-on: ubuntu-latest
+    container: ghcr.io/quick-lint/quick-lint-js-github-builder:v1
+    env:
+      C: clang-9
+      CFLAGS: "-stdlib=libc++"
+      CXX: clang++-9
+      CXXFLAGS: "-std=gnu++17 -stdlib=libstdc++ -O2"
```

Comment:
Is `-O2` necessary to make clang-tidy do its thing? If not, I recommend `-O0` (default) to speed up the build.